examples/advanced/job_config/hello-pt/simple_network.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
2024

Reply:
changed.

Reply:
has not changed though, but can be in next PR as well